README.md

```diff
@@ -2,152 +2,310 @@
+
+  <a href="https://twitter.com/dailynowco">
+    <img src="https://img.shields.io/twitter/follow/dailynowco?color=26A0ED&label=Follow&logo=twitter&logoColor=white&style=flat" alt="Twitter">
   </a>
+
   <a href="https://storybook.dailynow.co">
-    <img src="https://cdn.jsdelivr.net/gh/storybookjs/brand@master/badge/badge-storybook.svg" alt="Storybook">
+    <img src="https://img.shields.io/badge/%20-storybook-502ab0?logo=storybook&logoColor=white" alt="Storybook">
   </a>
+
+</p>
+
+
+
+
+## 🧶 Daily
+
+> Daily is an open source browser extension which provides curated dev news to your new tab.
+
+It gathers and ranks articles from tens of unique sources to help developers stay updated with the most relevant developer news. You can bookmark items, sync your data across devices, and read when you want. It works offline as well and has a progressive web app (PWA) for mobile.
+
+Daily is:
+
+* 🌟 **Well-maintained**: We are working continuously to introduce new features, fix bugs, and improve user experience — 40+ releases on average in a year.
+* ♾ **Constantly Updated**: Daily's article feed is constantly updated. As soon as new articles get published, they start appearing in Daily.
+* 🧵 **Open Source**: Daily is completely open source. You can see the code to learn how it works, and contribute to help it improve.
+
+<p align="center">
+    <img src="/assets/daily-header.jpg" alt="Daily" width="850">
+    <br>
+    <a href="https://www.youtube.com/watch?v=Oso6dYXw5lc"><strong>📺 Watch Intro Video → </strong></a>
+</p>
+
+## 📌 Get Daily
+
+Daily is currently available for Google Chrome, and Firefox. There's also a progressive web app (PWA) for mobile devices. Download for:
+
+<p align="center">
+    <a href="https://bit.ly/chromedaily">
+    <img src="https://img.shields.io/badge/%20-Chrome-red?logo=google-chrome&logoColor=white" alt="Download for Chrome" />
+    </a>
+    <a href="https://bit.ly/firefoxdaily">
+    <img src="https://img.shields.io/badge/%20-Firefox-orange?logo=mozilla&logoColor=white" alt="Download for Firefox" />
+    </a>
+    <a href="http://go.dailynow.co/">
+    <img src="https://img.shields.io/badge/%20-Mobile-502ab0" alt="Download for Mobile" />
+    </a>
 </p>
 
-Hi there 👋! So you probably here because you want to understand deeper what's going on behind the scenes of Daily... This is exactly the place for you so calm down and keep on reading 📖!
+## 📯 Philosophy
 
-## Don't know Daily!?
+We, as developers, spend a lot of time looking for valuable articles and blog posts. It's hard to catch up with all the latest happenings — coz it's frustrating and consumes a ton of time.
 
-Daily is a dev news curator delivering relevant updates to your new tab. It gathers and ranks articles from tens of different sources to help you optimize your time and boost your knowledge.
+That's why built Daily to help you:
 
+* 👨‍💻 Stay up-to-date
+* ⏳Save time
+* 📱Read anywhere
+* 🔖 Read when you want
```

Comment:
I would mention something about the passive consumption of content. This is one of our unique points, as a user you just have to open a new tab. You don't need to "want reading news".